test/testutils.jl

```diff
@@ -0,0 +1,7 @@
+function testmainfunction(pardict,seed,datapath)
+    println(pardict)
```

Comment:
print statements are not needed. They just pollute the developer's REPL when running the tests.